src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -964,10 +964,17 @@ export const GlobalProvider = memo(
                 const setInputSize = (size: Readonly<Size>) => {
                     modifyNode(id, (old) => {
                         const nodeCopy = copyNode(old);
-                        nodeCopy.data.inputSize = {
+                        const newInputSize: Record<string, Size> = {
```

Comment:
They may all be strings under the hood, but let's pretend that there's more structure to it.

```suggestion
                        const newInputSize: Record<InputId, Size> = {
```

Reply:
I'll commit that but I'm guessing it'll error. Maybe not though, since ultimately i did end up changing the way i did this after fixing that issue

Reply:
Argh, you're right. Object.entries returns strings. Yeah, my suggestion was wrong, it has to be string. My ad.